src/main/java/org/folio/helper/PurchaseOrderLineHelper.java

```diff
@@ -383,7 +380,7 @@ public String buildNewPoLineNumber(PoLine poLineFromStorage, String poNumber) {
    */
   public void updateEstimatedPrice(CompositePoLine compPoLine) {
     Cost cost = compPoLine.getCost();
-    cost.setPoLineEstimatedPrice(calculateEstimatedPrice(cost).getNumber().doubleValue());
+    cost.setPoLineEstimatedPrice(HelperUtils.calculateEstimatedPrice(cost).getNumber().doubleValue());
```

Comment:
This change is making code longer from my perspective. Previous version is more easier to read. Please consider in other places as well

Reply:
done